### PR TITLE
Add list of admins and board members

### LINKED ISSUE #2: Add board members and admin list
Add board members and admin list, in separate files from the charter.

### PR DESCRIPTION
**Pull request by nathanshammah**
This is taking a first stab at it. Updates on the QuTiP website will give more details, e.g., affiliations, biographic info, etc.

### CODE REVIEW

File: admins.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ # QuTiP Admin Team
+
+The functioning of the board of the QuTiP project can be found in the [governance document](governance.md#2-admin-team). The current members of the board are:
```
Comment (BoxiLi) [inline admins.md]:
```suggestion
The functioning of the admin team of the QuTiP project can be found in the [governance document](governance.md#2-admin-team). The current members of the admin team are:
```

File: admins.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Shahnawaz Ahmed
+- Eric Giguere
+- Neill Lambert
+- Boxi Li
+- Jake Lishman
+- Alex Pitchford
+- Nathan Shammah
```
Comment (BoxiLi) [inline admins.md]: Add Simon?

Comment (nathanshammah) [reply admins.md]: This has not been voted yet, so the admin list now is correct. Can be updated later with another PR to reflect eventual updates :).

**Review by BoxiLi — APPROVED** [review]

File: admins.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+ # QuTiP Admin Team
+
+The functioning of the admin team of the QuTiP project can be found in the [governance document](governance.md#2-admin-team). The current members of the admin team are:
+
+- Shahnawaz Ahmed
+- Eric Giguere
```
Comment (jakelishman) [inline admins.md]: is Eric's name missing some diacritics?

Comment (Ericgig) [reply admins.md]: Eric Gigu**è**re, but I don't really mind.

**Review by Ericgig — APPROVED** [review]

File: admins.md (hunk 3 above)
Comment (jakelishman) [reply admins.md]: we may as well do it right, I suppose!

**Review by jakelishman — APPROVED** [review]

**Comment by nathanshammah** [comment]
Thanks for spotting it @jakelishman and sorry @Ericgig.

**Review by hodgestar — APPROVED** [review]

**Comment by hodgestar** [comment]
@nathanshammah Approved and I added myself to the list of admins.

**Pull request merged**